src/app/lib/device/ext-player.js

```diff
@@ -14,10 +14,35 @@
             name: i18n.__('External Player'),
         },
 
-        play: function (streamModel) {
-            // "" So it behaves when spaces in path
+        play: function(streamModel) {
+            var options = {};
+            var args = [];
             var url = streamModel.attributes.src;
-            var cmd = path.normalize('"' + this.get('path') + '" ') + getPlayerSwitches(this.get('id')) + ' ';
+            var cmd;
+
+            if (process.platform == 'win32') {
+                // "" So it behaves when spaces in path
+                //Node.js Format: spawn('"with spaces.cmd"', ['arg with spaces'], { shell: true });
+                cmd = path.normalize('"' + this.get('path') + '"');
+                options = {
+                    shell: true
+                };
+
+            } else
+            if (process.platform == 'linux') {
```

Comment:
Could you please change this to:
`if (process.platform == 'linux' || process.platform == "darwin")`
That way it will work on OsX as well.